### DISCUSSION TITLE
no save to disc method?

### DISCUSSION THREAD

**Discussion by Zawardo1000** [mjs/imapclient#605] [Q&A]
I Need to save .eml, but i have not foud any suitable method in the imapclient class.
Have i overseen, or Is It something in the todo list?
Thanks a lot!

**Comment by mjs** [comment]
Doing a `RFC822` fetch should give you the same data as what is typically found in a .eml file. 

There's an example in the docs here: https://imapclient.readthedocs.io/en/3.0.0/concepts.html#working-with-fetched-messages
